backend/services/members/tests/mutations.spec.ts

```diff
@@ -68,58 +130,95 @@ describe('[Mutations]', () => {
   })
 
   beforeEach(() => {
-    // Due to an error in @types/chai-spies ts-ignore is necessary.
-    // Pull request solving the issue https://github.com/DefinitelyTyped/DefinitelyTyped/pull/49678§
-    // @ts-ignore
+    sandbox.on(dataSources.memberAPI, 'createMember', (input) => new Promise(resolve => resolve(member)))
+    sandbox.on(dataSources.memberAPI, 'updateMember', (id, input) => new Promise(resolve => resolve(member)))
+    sandbox.on(dataSources.memberAPI, 'removeMember', (id) => new Promise(resolve => resolve(member)))
     sandbox.on(dataSources.positionAPI, 'createPosition', (ctx, input) => new Promise(resolve => resolve(input.name ? true : false)))
-    // @ts-ignore
     sandbox.on(dataSources.positionAPI, 'updatePosition', (ctx, id, input) => new Promise(resolve => resolve(id ? true : false)))
-    // @ts-ignore
     sandbox.on(dataSources.positionAPI, 'removePosition', (ctx, id) => new Promise(resolve => resolve(id ? true : false)))
-    // @ts-ignore
     sandbox.on(dataSources.committeeAPI, 'createCommittee', (ctx, input) => new Promise(resolve => resolve(input.name ? true : false)))
-    // @ts-ignore
     sandbox.on(dataSources.committeeAPI, 'updateCommittee', (ctx, id, input) => new Promise(resolve => resolve(id ? true : false)))
-    // @ts-ignore
     sandbox.on(dataSources.committeeAPI, 'removeCommittee', (ctx, id) => new Promise(resolve => resolve(id ? true : false)))
   })
 
   afterEach(() => {
     sandbox.restore();
   })
 
+  describe('[member]', () => {
+
+    it('try creating member no user', async () => {
+      const { errors } = await client.mutate({ mutation: CREATE_MEMBER });
+      expect(errors).to.exist
+    })
+
+    it('creates a member', async () => {
+      const { server, dataSources } = constructTestServer({user: {keycloak_id: 'kc_1'}});
+      const { mutate } = createTestClient(server);
+      sandbox.on(dataSources.memberAPI, 'createMember', (input) => new Promise(resolve => resolve(member)));
+      const { data } = await mutate({ mutation: CREATE_MEMBER });
+      expect(data.member.create).to.deep.equal(member);
+    })
+
+    it('try updating member no user', async () => {
```

Comment:
Namngivningen på de här följer inte riktigt samma mönster 

typ "throws error when user is not signed in"